src/control/server/ctl_storage_rpc.go

```diff
@@ -500,7 +500,9 @@ func (cs *ControlService) getMetaClusterCount(engineCfg *engine.Config, devToAdj
 
 	if dev.GetRoleBits()&storage.BdevRoleMeta != 0 {
 		// TODO DAOS-14223: GetMetaSize() should reflect custom values derived from pool
-		//                  create --meta-size or --mem-size options.
+		//                  create --meta-size or --mem-ratio options. Currently this is
+		//                  just VOS file size. Adjustment should also consider that targets
+		//                  will be distributed across all SSDs in given tier.
```

Comment:
the question for me still is how do we factor in that the meta blobs will not all be applied on the same SSD if there are multiple across the tier @knard-intel have any ideas?